### DISCUSSION TITLE
Compatibility check Golmar T-840C

### DISCUSSION THREAD

**Discussion by Orbi-Code** [PricelessToolkit/ESPBell-LITE#10] [Q&A] [answered]
Can someone confirm if this intercom is compatible with the espbell?

How to wire it?

It's a 4+n connection from the Platea series, Series 90 bus technology.

![ENG_REV0100_T110.pdf](https://github.com/PricelessToolkit/ESPBell-LITE/files/14879563/ENG_REV0100_T110.pdf)

**Comment by PricelessToolkit** [comment]
Hi, probably yes, the relay needs to be connected to ( - and HZ ) but the doorbell... I am not sure! maybe between ( "+ and SA").

**Reply by Orbi-Code** [reply]
Can you please show/tell me which colors of wires go to which connection on my intercom?

**Reply by PricelessToolkit** [reply, marked as answer]
Wires are labeled on the back side of the PCB, (LOCK wires are doubled!!!! two pairs)

<img src="https://raw.githubusercontent.com/PricelessToolkit/ESPBell-LITE/main/img/intercom_wiring.jpg"/>

1. Two of the "LOCK" wires to "-" the second pair of "LOCK" wires to "HZ"
2.  One of the "BELL" wires to "+" and the second "BELL" wire to "SA" (I'm not sure if you have a multimeter you can check it).
3.  For power "GND" to "-" of the power supply, and the "VCC" to the "+" of power supply "5V MAX"
